chart/jenkins-operator/templates/operator.yaml

```diff
@@ -26,7 +26,7 @@ spec:
           image: {{ .Values.operator.image }}
           imagePullPolicy: {{ .Values.operator.imagePullPolicy }}
           ports:
-            - name: http
+            - name: {{ .Values.portName }}
```

Comment:
In the original [issue](https://github.com/jenkinsci/kubernetes-operator/issues/736) the request was to have the possibility of specifying the name of the port of the result jenkins pod not the operator.
This will require to add a serviceName [here](https://github.com/jenkinsci/kubernetes-operator/blob/d91a729ccf895a9e10155ccc262e1f1d826ad4af/api/v1alpha2/jenkins_types.go#L386) and more golang code, finally the helm chart change.
If you are looking to something more easy to do I can suggest you to take a look on the others `good-first-issue` issue :)
If don't you can start with some changes and we can double check the code together :)

Reply:
I am new to all this but I'll try to resolve this issue as soon as I can and get back to you. Thank you for guiding me in this, I really appreciate that.

Reply:
@brokenpip3 Is that right?